integration-tests/hibernate-orm-spatial/pom.xml

```diff
@@ -0,0 +1,157 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>quarkus-integration-tests-parent</artifactId>
+        <groupId>io.quarkus</groupId>
+        <version>999-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>quarkus-integration-test-hibernate-orm-spatial</artifactId>
+    <name>Quarkus - Integration Tests - Hibernate ORM with Spatial</name>
+    <dependencies>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-rest</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-hibernate-orm</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.hibernate.orm</groupId>
+            <artifactId>hibernate-spatial</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.geolatte</groupId>
+            <artifactId>geolatte-geom</artifactId>
+            <version>1.10</version>
+        </dependency>
```

Comment:
This shouldn't be necessary... ? Geolatte is a transitive dependency of `hibernate-spatial` already.

If you really want to do this because you think every JAR we directly use in an app should be declared in dependencies (which is fair), then we'll need to find a way to make sure the version of this dependency doesn't get out of date, and stays aligned with whatever Hibernate ORM depends on. Which is doable, but I'm not sure it's worth the trouble.